File: app/src/main/java/com/artmurka/artmurkaapp/model/pojo/itemlist/good/EntryPriceOld.java

package com.artmurka.artmurkaapp.model.pojo.itemlist.good;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EntryPriceOld {

    @SerializedName("price_raw")
    @Expose
    private Integer priceRaw;
    @SerializedName("price")
    @Expose
    private Integer price;

    public Integer getPriceRaw() {
        return priceRaw;
    }

    public void setPriceRaw(Integer priceRaw) {
        this.priceRaw = priceRaw;
    }

    public Integer getPrice() {
        return price;
    }

    public void setPrice(Integer price) {
        this.price = price;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/itemlist/EntryPriceOld.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EntryPriceOld {

    @SerializedName("price")
    @Expose
    var price: Int = 0
    @SerializedName("price_raw")
    @Expose
    var priceRaw: Int = 0

}
